tdcommon/src/main/java/thredds/server/catalog/builder/CatalogScanBuilder.java

```diff
@@ -22,9 +23,20 @@ public CatalogScanBuilder(DatasetBuilder parent, String name, String path, Strin
     this.path = path;
     this.location = location;
     this.watch = watch;
-    this.context = context;
+    this.context = StringUtil2.trim(context, '/');
   }
 
+  public CatalogScanBuilder(DatasetBuilder parent, CatalogScan from, String context) {
+    super(parent, from);
+    this.path = from.getPath();
+    this.location = from.getLocation();
+    this.watch = from.getWatch();
+    this.context = StringUtil2.trim(context, '/');
+  }
+
+  /**
+   * @deprecated Use {@link #CatalogScanBuilder(DatasetBuilder, CatalogScan, String)} instead
+   */
   public CatalogScanBuilder(DatasetBuilder parent, CatalogScan from) {
     super(parent, from);
```

Comment:
I'd have this method call the new one with "thredds" as the context" so that if the constructor ever changes it only has to change in one place.